web/packages/core/src/internal/player/inner.tsx

```diff
@@ -2328,3 +2324,31 @@ function parseAllowScriptAccess(
             return null;
     }
 }
+
+/**
+ * Detect direction (LTR/RTL) of the preferred language of the browser.
+ *
+ * @returns 'ltr' or 'rtl' depending on the detected direction
+ */
+function detectBrowserDirection(): string {
+    const browserLocale = new Intl.Locale(navigator.language) as any;
```

Comment:
Is the `as any` necessary. What happens without it?

Reply:
I think the code should works exactly the same without it.

Reply:
You're right, removed